### DISCUSSION TITLE
Auth::guard('api')->user()  can not be cached？

### DISCUSSION THREAD

**Discussion by williamqian** [laravel/framework#33276] [General]
everytime I run Auth::guard('api')->user() , it will run a sql like `SELECT * FROM `oauth_access_tokens` WHERE `id` = '79188bfc389b4e3416c15e5351f400e773126dc4bc3783a74d8000eb525e0a6950d695b0a6d31072' LIMIT 1`

**Comment by Cyrille37** [comment]
`Authenticatable::user()` querying the UserProvider while `Authenticatable::getUser()` return the currently cached user.

You can read the difference at `laravel/framework/src/Illuminate/Auth/SessionGuard.php`.
